enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/task/DocumentsVerificationSyncTask.java

```diff
@@ -40,7 +40,7 @@ public DocumentsVerificationSyncTask(VerificationProcessingBatchService verifica
      * Scheduled task to check documents verifications at the target provider
      */
     @Scheduled(cron = "${enrollment-server-onboarding.document-verification.checkDocumentsVerifications.cron:0/5 * * * * *}", zone = "UTC")
-    @SchedulerLock(name = "checkDocumentsVerifications", lockAtLeastFor = "1s", lockAtMostFor = "5m")
+    @SchedulerLock(name = "documentVerificationLock", lockAtLeastFor = "1s", lockAtMostFor = "5m")
```

Comment:
I am not a big fan of constants, but thinking about that, because this literal is important and used at several places.

Reply:
Good idea.